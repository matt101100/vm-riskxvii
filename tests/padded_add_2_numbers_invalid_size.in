tests/memory_image_binaries/padded_add_2_numbers_invalid_size.mi