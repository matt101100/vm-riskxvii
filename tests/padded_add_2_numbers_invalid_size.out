Invalid file path provided or error loading image.
